usaspending_api/disaster/v2/views/recipient/count.py

```diff
@@ -23,20 +22,33 @@ class RecipientCountViewSet(FabaOutlayMixin, AwardTypeMixin, DisasterBase):
 
     @cache_response()
     def post(self, request: Request) -> Response:
```

Comment:
Testing performance around this I am seeing it timeout against QAT. How was performance for you?